src/portable/credentials.rs

```diff
@@ -42,6 +62,7 @@ pub fn show_credentials(options: &Options, c: &ShowCredentials) -> anyhow::Resul
             ("Port", creds.port.to_string().as_str()),
             ("User", creds.user.as_str()),
             ("Password", creds.password.map(|_| "<hidden>").unwrap_or("<none>")),
+            ("Secret Key", secret_key.map(|_| "<hidden>").unwrap_or("<none>")),
```

Comment:
Can we show some basic parameters of the token (i.e. `<edgedb.cloud AWS token>`)? Or are all of them are encrypted?

Reply:
Yeah we can extract the token issuer (domain) and show that